docs/_tutorials/getting_started.md

```diff
@@ -176,7 +176,8 @@ const app = new App({
   token: process.env.SLACK_BOT_TOKEN,
   signingSecret: process.env.SLACK_SIGNING_SECRET,
   socketMode: true,
-  appToken: process.env.SLACK_APP_TOKEN
+  appToken: process.env.SLACK_APP_TOKEN,
+  port: process.env.PORT || 3000 // socket mode doesn't listen on a port, but in case you want your app to respond to OAuth, you still need to listen on some port!
```

Comment:
Yeah, something like this may be easier to read

```suggestion
  // Socket Mode doesn't listen on a port, but in case you want your app to respond to OAuth,
  // you still need to listen on some port!
  port: process.env.PORT || 3000
```